samplers/jaegerremote/sampler_remote_options.go

```diff
@@ -37,6 +41,45 @@
 	logger                  logr.Logger
 }
 
+func getEnvOptions() ([]Option, []error) {
+	var options []Option
+	// list of errors which will be logged once logger is set by the user
+	var errs []error
+
+	args := strings.Split(os.Getenv("OTEL_TRACES_SAMPLER_ARG"), ",")
+	for _, arg := range args {
+		keyValue := strings.Split(arg, "=")
+		if len(keyValue) != 2 {
+			errs = append(errs, fmt.Errorf("argument %s is not of type '<key>=<value>'", arg))
+			continue
+		}
+		key := strings.Trim(keyValue[0], " ")
+		value := strings.Trim(keyValue[1], " ")
+
+		switch key {
+		case "endpoint":
+			options = append(options, WithSamplingServerURL(value))
+		case "pollingIntervalMs":
+			intervalMs, err := strconv.ParseUint(value, 10, 0)
+			if err != nil {
+				errs = append(errs, fmt.Errorf("%s parsing failed with :%w", key, err))
+				continue
+			}
+			options = append(options, WithSamplingRefreshInterval(time.Duration(intervalMs)*time.Millisecond))
```

Comment:
I can fix this with parsing as int64. But this allows the possibility of passing a negative `intervalMs` as a refresh interval. Should I implement a logic to block this? @yurishkuro

Reply:
up to you. Strictly speaking the values should be checked in the constructor of the sampler, since it can still receive out-of-range parameters, so validation can be at the latest possible stage.

Reply:
Makes sense. I have parsed this as int without negative integer validation.